docs/proposals/05-etcd-operator-tasks.md

```diff
@@ -13,46 +13,6 @@ reviewers:
 
 # DEP-05: Operator Out-of-band Tasks
 
-## Table of Contents
```

Comment:
If you're removing the Table of Contents, please also remove it from the [proposal template](https://github.com/gardener/etcd-druid/blob/master/docs/proposals/00-template.md).

Reply:
Done